src/openrct2/drawing/Drawing.h

```diff
@@ -648,24 +649,23 @@ void FASTCALL gfx_draw_sprite_raw_masked_software(
 // string
 void gfx_draw_string(rct_drawpixelinfo* dpi, const_utf8string buffer, uint8_t colour, const ScreenCoordsXY& coords);
 
+/** @deprecated */
```

Comment:
You could use https://en.cppreference.com/w/cpp/language/attributes/deprecated, this will add some compiler warnings but will actually make it clear that its not supposed to be used anymore.

Reply:
The problem with that is that compilation will fail if a compiler warning is output in our project.

Reply:
Is there no way to ignore this specific warning maybe?

Reply:
I don't know, but I'm sure @janisozaur does.

Reply:
I mean its not that big of a deal, we are going to remove them anyway soonish I guess.